File: SpringMicroservices/microservice-music/src/main/java/com/microservice/music/infrastructure/adapters/in/rest/dto/AlbumDto.java
package com.microservice.music.infrastructure.adapters.in.rest.dto;

import lombok.*;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class AlbumDto {
    private Long idAlbum;
    private String title;
    private String description;
    private String imageCover;
    private List<SongDto> songs;
    private List<Long> songIds;
    private String userId;
}

File: ProyFronted/app/src/main/java/com/example/proyfronted/backend/Music/Dto/AlbumDto.kt
package com.example.proyfronted.backend.Music.Dto




data class AlbumDto(
    val idAlbum: Long? = null,
    val title: String,
    val description: String? = null,
    val imageCover: String? = null,
    val visible: Boolean = true,
    val songs: List<SongDto>? = null,
    val songIds: List<Long> = emptyList(),
    val userId: String? = null
)
